lib/std/comptime_string_map.zig

```diff
@@ -71,8 +72,8 @@ pub fn ComptimeStringMapWithEql(
         };
         mem.sortUnstableContext(0, sorted_kvs.len, SortContext{ .kvs = &sorted_kvs });
 
-        const min_len = sorted_kvs[0].key.len;
-        const max_len = sorted_kvs[sorted_kvs.len - 1].key.len;
+        const min_len = if (empty_list) -1 else sorted_kvs[0].key.len;
```

Comment:
Could you instead break out of the block after line 53 with the empty list?

Reply:
Ah, yeah I can.  I was originally trying to preserve the type of the result object, but given this is comptime, the empty case can be represented by an empty object.  Fixed!